src/components/Footer/Footer.tsx

```
@@ -0,0 +1,67 @@
+"use client";
+import { soundState } from "@/atoms/SoundAtom";
+import { Icon } from "@iconify/react";
+import { getAnalytics, logEvent } from "firebase/analytics";
+import { motion } from "framer-motion";
+import React from "react";
+import { Link } from "react-scroll";
+import { useRecoilValue } from "recoil";
+import useSound from "use-sound";
+import styles from "../../styles/components/Footer.module.scss";
+type FooterProps = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)